src/main.ts

```diff
@@ -0,0 +1,5 @@
+import { createApp } from 'vue'
+import './style.css'
+import App from './App.vue'
+
+createApp(App).mount('#app')
```

Comment:
Is `mount()` Vue's equivalent of React's `render()` method? That's a pretty cool implementation!

Reply:
I think in this case, `createApp().mount()` in Vue is closer to `ReactDOM.createRoot()` in React

More info [here](https://vuejs.org/api/application.html#createapp) and [here](https://vuejs.org/api/application.html#app-mount) if interested